modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/DataRegionMetricsImpl.java

```diff
@@ -114,8 +115,10 @@
      * @param dataRegionMetricsProvider Data region metrics provider.
      */
     public DataRegionMetricsImpl(DataRegionConfiguration memPlcCfg,
-        GridMetricManager mmgr,
-        DataRegionMetricsProvider dataRegionMetricsProvider) {
+                                 GridMetricManager mmgr,
```

Comment:
1. Not used log.
2. Looks like arguments formatted not by Ignite / GG code style.